docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<p>Annotation -&gt; Cell-type annotation? </p><p> </p><p>I also started wondering if this should become a tutorial on it's own as it's quite comprehensive and shows multiple ways of doing so. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>